lua/orgmode/utils/treesitter.lua

```diff
@@ -53,7 +52,7 @@ function M.set_node_text(node, text, front_trim)
       ec = ec + 1
     end
   end
-  vim.api.nvim_buf_set_text(0, sr, sc, er, ec, { text })
+  pcall(vim.api.nvim_buf_set_text, 0, sr, sc, er, ec, lines)
```

Comment:
I pcall this because there are cases where the old node is bigger than the new node, so the columns don't match. Neovim still handles this correctly, but raises an error.
I'd be nicer if it would have the `strict_indexing` flag like `nvim_buf_set_lines`